resources/runTests.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *  Copyright (c) Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { exec } = require('./util');
+
+exec(
+  'jest',
+  '--config',
+  '../../jest.config.js',
```

Comment:
Use a path relative to this script, i.e. `path.join(__dirname, '..', 'jest.config.js')`